webapp/src/components/call_error_modal.tsx

```diff
@@ -117,6 +119,10 @@ export const CallErrorModal = (props: Props) => {
             onHide={() => dispatch(clearClientError())}
             handleConfirm={onConfirm}
             contentPadding={'48px 32px'}
+            components={{
+                Header: Header as any,
```

Comment:
@cpoile Wrapping `Modal.Header` using styled components seems to break the interface and cause issues. I don't particularly like this but after fighting with typescript for a good 20 minutes I just gave up. Please let me know if you have ideas on how to properly deal with this.